2 dz/4/functions.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include <cmath>
+#include <iomanip>
```

Comment:
I'm not sure here. Do we need to include these libraries here? Or it's a part of the implementation and can be put into the .cpp file?
We make only declarations in the header file.